include/flatbuffers/minireflect.h

```diff
@@ -332,6 +332,31 @@ inline std::string FlatBufferToString(const uint8_t *buffer,
   return tostring_visitor.s;
 }
 
+inline std::map<std::string, std::string>
```

Comment:
These functions look very wasteful, allocating memory from what are otherwise nice static structures. The list of attributes is typically very short so linear search on them should be ok, or if not, we can make sure they're always written in sorted order so we can optionally do binary search.